docs/user_guide/1_arm_and_hand.md

```diff
@@ -0,0 +1,659 @@
+# UR Arm and Dexterous Hand 
+
+This documentation describes the setup of a Shadow Dexterous Hand attached to a UR arm (UR5 or UR10).
+
+```eval_rst
+.. image:: ../img/UR10_hand_E.jpeg
+  :width: 600
+  :align: left
+```
+
+## Setting up a real system 
+
+### Setting up the arm
+
+1. Unpack the robot arm and the control box.
+2. Mount the robot on a sturdy surface strong enough to withstand at least 10 times its weight.
+3. Place the control box on its foot.
+4. Plug on the robot cable between the robot and the control box.
+5. Plug in the mains plug of the control box.
+
+To quickly start up the robot after it has been installed, perform the following steps:
+1. Press the power button on the teach pendant.
+2. Wait a minute while the system is starting up, displaying text on the touch screen.
+3. Press ON button on the touch screen. Wait a few seconds until robot state changes to idle.
+4. Press START button on the touch screen. The robot now makes a sound and moves a little while releasing the brakes.
+
+#### Configuring the network
+
+In order to use the robot with our driver you need to change the network setup of the robot via the pedant, performing the following steps:
```

Comment:
Typo, pedant -> pendant.

Reply:
Or is this too pedantic of me? I'll see myself out...

Reply:
:P

Reply:
Done